README.md

```diff
@@ -416,321 +412,20 @@ Python | 3.6 or higher | No | *N/A* | *Pre-installed or Installed by user* | [PS
-- `dpc++` (default) for the [Intel(R) oneAPI DPC++ Compiler](https://software.intel.com/en-us/oneapi/dpc-compiler) and for the `clang++` from [Intel project for LLVM* technology](https://github.com/intel/llvm/releases) compilers. 
-- `hipsycl` for the [hipSYCL](https://github.com/illuhad/hipSYCL) SYCL implementation
-
-All options specified in the Conan section are available to CMake. You can specify these options using `-D<cmake_option>=<value>`.
-
-The following table provides a detailed mapping of options between Conan and CMake.
-
-Conan Option | CMake Option | Supported Values | Default Value 
- :---------- | :----------- | :--------------- | :---          
-build_shared_libs        | BUILD_SHARED_LIBS        | True, False         | True      
-enable_mklcpu_backend    | ENABLE_MKLCPU_BACKEND    | True, False         | True      
-enable_mklgpu_backend    | ENABLE_MKLGPU_BACKEND    | True, False         | True      
-*Not Supported*          | ENABLE_CUBLAS_BACKEND    | True, False         | False     
-*Not Supported*          | ENABLE_CUSOLVER_BACKEND  | True, False         | False     
-*Not Supported*          | ENABLE_CURAND_BACKEND    | True, False         | False     
-*Not Supported*          | ENABLE_NETLIB_BACKEND    | True, False         | False     
-*Not Supported*          | ENABLE_ROCBLAS_BACKEND   | True, False         | False     
-enable_mklcpu_thread_tbb | ENABLE_MKLCPU_THREAD_TBB | True, False         | True      
-build_functional_tests   | BUILD_FUNCTIONAL_TESTS   | True, False         | True      
-build_doc                | BUILD_DOC                | True, False         | False     
-target_domains (list)    | TARGET_DOMAINS (list)    | blas, lapack, rng   | All domains 
-
-*Note: `build_functional_tests` and related CMake option affects all domains at a global scope.*
-
-*Note: When building with hipSYCL, `-DHIPSYCL_TARGETS` additionally needs to be provided according to the targeted hardware. For the options, see the tables in the hipSYCL specific sections*
-
----
-
-## Project Cleanup
-
-Most use-cases involve building the project without the need to cleanup the build directory. However, if you wish to cleanup the build directory, you can delete the `build` folder and create a new one. If you wish to cleanup the build files but retain the build configuration, following commands will help you do so. They apply to both `Conan` and `CMake` methods of building this project.
-
-```sh
-# If you use "GNU/Unix Makefiles" for building,
-make clean
-
-# If you use "Ninja" for building
-ninja -t clean
-```
+## Documentation
+- [Contents](https://oneapi-src.github.io/oneMKL/)
+- [About](https://oneapi-src.github.io/oneMKL/introduction.html)
+- [Get Started](https://oneapi-src.github.io/oneMKL/selecting_a_compiler.html)
+- [Developer Reference](https://oneapi-src.github.io/oneMKL/onemkl-datatypes.html)
```

Comment:
The link does not seem correct for Developer Reference. Also, what does Developer Reference mean in this repo? Are you referring to oneMKL product developer reference?

Reply:
It refers to the rST documentation on GitHub Pages for oneMKL Interfaces.

Reply:
For developer reference, I would expect the link to a more top level page - rather than onemkl-datatypes.html?